content/podcast/2023/ep10_flux_transcripts_errors.md

```diff
@@ -0,0 +1,44 @@
+title=The Flux executor, bytesize transcripts and improved error messaging
+episode=10
+description=The Flux executor, bytesize transcripts and improved error messaging.
+date=2023-01-18
+type=podcast
+subtype=News and Views
+youtubeid=5UDIGQVo9hA
+image=img/podcast_ep10.jpg
+tags=news and views,opensource,community
+status=published
+author= Developer advocates
+icon=logo_podcast_channels.jpg
+~~~~~~
+
+In this News and Views episode, [Chris Hakkaart](https://twitter.com/chris_hakk) and [Marcel Ribeiro-Dantas](https://twitter.com/mribeirodantas) discuss the hottest topics in the Nextflow world.
+
+<!-- end-archive-description -->
+
+#### The Flux executor
+
+- [Flux](https://flux-framework.org/) is a job manager akin to Slurm, Sun Grid Engine, or similar and has been added to Nextflow as an executor.
+- Flux was originally designed as an HPC job manager but is being extended it to more cloud environments and will be one to watch in the future.
+- A great example of a feature that was added to Nextflow by the community [nextflow-io/nextflow#3412](https://github.com/nextflow-io/nextflow/pull/3412).
+
+#### Transcripts for bytesize seminars
+
+- Video transcripts create a better user experience by increasing accessibility for users who are deaf or hard of hearing, improving audience comprehension, and enabling flexible viewing in sound-sensitive environments.
+- [Franziska Bonath](https://github.com/FranBonath) has been working towards making the [nf-core bytesize](https://nf-co.re/events) seminars more accessible by adding transcripts that can be used for subtitles to all videos. 
+- All transcripts for all bytesize seminars have now been made and are now under review.
```

Comment:
I think we should mention they can be read as a full text at the bytesize page in the nf-core website. What do you think? Example: https://nf-co.re/events/2022/bytesize_nftest